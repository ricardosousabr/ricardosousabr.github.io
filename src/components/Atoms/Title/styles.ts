import styled from "styled-components";

export const TextStyle = styled.h2`
  color: white;
  font-size: 20px;
  line-height: 35px;
`