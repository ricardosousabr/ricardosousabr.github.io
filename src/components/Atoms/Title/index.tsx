import { TextStyle } from "./styles";
import { TitleProps } from "./types";

export default function Title({children}: TitleProps) {
  return <TextStyle>{ children }</TextStyle>
};
