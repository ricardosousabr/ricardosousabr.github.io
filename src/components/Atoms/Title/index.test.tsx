import renderer from "react-test-renderer";
import Title from "./index";

test("Link renders correctly", () => {
  const tree = renderer.create(<Title>Click</Title>).toJSON();
  expect(tree).toMatchSnapshot();
});
