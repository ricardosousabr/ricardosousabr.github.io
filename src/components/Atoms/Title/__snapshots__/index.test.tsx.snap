// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link renders correctly 1`] = `
<h2
  className="styles__TextStyle-sc-mjcf9p-0 hEVqIA"
>
  Click
</h2>
`;
